agenta-backend/agenta_backend/routers/app_variant.py

```diff
@@ -167,10 +167,12 @@ async def start_variant(
         if settings.feature_flag == "demo":
             if not settings.openai_api_key.startswith("sk-"):
                 raise HTTPException(
-                    status_code=404,
+                    status_code=403,
```

Comment:
I d go for 400 instead of 403. 403 means forbidden.
400 is bad request, I believe it fits better the context to indicate that the key is in the wrong format.

Reply:
Alright, thank you.